turte_graphics/turtle.js

```diff
@@ -0,0 +1,111 @@
+class Turtle {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+    this.position = [x, y];
+    this.path = [[x, y]];
+    this.direction = "x";
```

Comment:
After seeing how `left()` and `right()` work, I think I would try storing the direction as an angle rather than as a string). I.e. 
```
"x" -> 0
"y" -> 90
"-x" -> 180
"-y" -> 270
```
I think the `left`/`right` methods would then be a lot simpler (since you would just have to add/subtract 90 and check if it's `< 0` or `>=360` to "reset" it).
```

Reply:
done! thanks for explaining